### DISCUSSION TITLE
Why my commit activity can not show at my profile？

### DISCUSSION THREAD

**Discussion by shchen-Lab** [community/community#57208] [Discussions] [answered]
### Select Topic Area

Bug

### Body
Why my commit activity can not show at my profile？  This an bug or setting problem？ How to fix this？

<img width="479" alt="75dfdcc17d669b58b7585c7f0ed3e0b" src="https://github.com/community/community/assets/75290921/980ecba2-f2e0-488c-96af-abdd82d8b55e">
<img width="868" alt="de9e5e9b9be69df125fefd7466581e6" src="https://github.com/community/community/assets/75290921/2222683b-ab3c-468e-8e15-6f3d094c91c7">
![image](https://github.com/community/community/assets/75290921/1442636a-0f18-48a2-a856-7c96bdae9d01)

**Comment by airtower-luna** [comment, marked as answer]
Commits in a fork won't show in your contribution graph until they're merged into the upstream repository.

See also: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile
